assets/large_language_models/rag/environments/rag/asset.yaml

```
@@ -0,0 +1,6 @@
+name: llm-rag
+version: 0.10.0
```

Would you please set version to "auto" for environments? This will use one-up version numbers for environment versions and image tags.
```suggestion
version: auto
```

---

Awesome - thank you for the suggestion.